othello/moderator/runners.py

```diff
@@ -10,13 +10,27 @@
 
 from ..sandboxing import get_sandbox_args
 from .utils import ServerError, UserError, capture_generator_value
+from .moderator import Player
+
+LEGACY_MOVES = {11: 0, 12: 1, 13: 2, 14: 3, 15: 4, 16: 5, 17: 6, 18: 7, 21: 8, 22: 9, 23: 10, 24: 11, 25: 12, 26: 13, 27: 14, 28: 15, 31: 16, 32: 17, 33: 18, 34: 19, 35: 20, 36: 21, 37: 22, 38: 23, 41: 24, 42: 25, 43: 26, 44: 27, 45: 28, 46: 29, 47: 30, 48: 31, 51: 32, 52: 33, 53: 34, 54: 35, 55: 36, 56: 37, 57: 38, 58: 39, 61: 40, 62: 41, 63: 42, 64: 43, 65: 44, 66: 45, 67: 46, 68: 47, 71: 48, 72: 49, 73: 50, 74: 51, 75: 52, 76: 53, 77: 54, 78: 55, 81: 56, 82: 57, 83: 58, 84: 59, 85: 60, 86: 61, 87: 62, 88: 63}
```

Comment:
If we're going to have this, I'd prefer it to be dynamically generated (perhaps by a dictionary comprehension) so it's clearer that it's actually correct.

Reply:
Its just a simple lookup table, converting this to a comprehension would be more complicated. It may not even be possible to do in a comprehension.

Reply:
made it a comprehension